app/src/main/java/fr/free/nrw/commons/upload/categories/CategoriesPresenter.kt

```diff
@@ -79,8 +81,13 @@ class CategoriesPresenter
                         },
                     ),
             )
+
+            //isInitialLoad = false
```

Comment:
Remove commented code?

Reply:
I have removed the suggested line. Please take another look when you have time. Thank you for your feedback!